tools/obscuroscan/static/index.js

```diff
@@ -35,8 +34,6 @@ const initialize = () => {
     }, 1000);
 
     document.getElementById(idFormGetRollup).addEventListener(typeSubmit, async (event) => {
-        event.preventDefault();
```

Comment:
Removed because we actually want the form to perform its action (jumping down the page) now.